### PR TITLE
change legacy FSE tag to prevent WP 5.8 issues

### PR DESCRIPTION
**Pull request by mattwiebe**
#### Changes proposed in this Pull Request:

Change the legacy FSE theme tag from `full-site-editing` to `wpcom-fse` for Varia-based themes.

#### Related issue(s):
https://github.com/Automattic/wp-calypso/issues/54922

Should supersede https://github.com/Automattic/themes/pull/4335

### CODE REVIEW

**Comment by mattwiebe** [comment]
Ah shoot, caught some legit FSE themes in that

File: mayland-blocks/style.css [hunk 1]
```diff
@@ -12,7 +12,7 @@ License: GNU General Public License v2 or later
 License URI: https://raw.githubusercontent.com/Automattic/themes/trunk/LICENSE
 Template: blockbase
 Text Domain: mayland-blocks
-Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, featured-images, full-site-editing, block-patterns, rtl-language-support, theme-options, threaded-comments, translation-ready, wide-blocks
+Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, featured-images, wpcom-fse, block-patterns, rtl-language-support, theme-options, threaded-comments, translation-ready, wide-blocks
```
Comment (jffng) [inline mayland-blocks/style.css]: I think this theme should keep the `full-site-editing` tag?

**Review by jffng — COMMENTED** [review]
Just one comment, other than that LGTM.

File: mayland-blocks/style.css (hunk 1 above)
Comment (mattwiebe) [reply mayland-blocks/style.css]: Yes it should, thanks.

**Review by jffng — APPROVED** [review]

**Pull request merged**